email-alert-api/docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: '3.7'
+
+x-email-alert-api: &email-alert-api
+  build:
+    context: .
+    dockerfile: email-alert-api/Dockerfile
+  image: email-alert-api
+  volumes:
+    - ..:/govuk:delegated
+    - bundle:/usr/local/bundle
+  working_dir: /govuk/email-alert-api
+
+services:
+  email-alert-api-default:
+    <<: *email-alert-api
+    depends_on:
+      - postgres
+      - redis
+    environment:
+      DATABASE_URL: "postgresql://postgres@postgres/email-alert-api"
+      TEST_DATABASE_URL: "postgresql://postgres@postgres/email-alert-api-test"
+      REDIS_URL: redis://redis
+
+  email-alert-api-backend: &email-alert-api-backend
+    <<: *email-alert-api
+    depends_on:
+      - postgres
+      - redis
+    environment:
+      DATABASE_URL: "postgresql://postgres@postgres/email-alert-api"
+      REDIS_URL: redis://redis
+      VIRTUAL_HOST: email-alert-api.dev.gov.uk
+      HOST: 0.0.0.0
+    ports:
+      - "3000"
+    command: bin/rails s -P /tmp/rails.pid
+
+  email-alert-api-worker:
+    <<: *email-alert-api
+    depends_on:
+      - postgres
+      - redis
+    environment:
+      DATABASE_URL: "postgresql://postgres@postgres/email-alert-api"
+      REDIS_URL: redis://redis
+      VIRTUAL_HOST: email-alert-api.dev.gov.uk
+      HOST: 0.0.0.0
+    ports:
+      - "3000"
+    command: bundle exec sidekiq -C ./config/sidekiq.yml
```

Comment:
I'd suggest we still have an e2e entry for consistency, however I think the only additional dependency that would have to backend is email-alert-api-worker.